libs/blocks/ost/ost.js

```diff
@@ -19,6 +22,55 @@ if (token) {
 
 document.body.classList.add('tool', 'tool-ost');
 
+async function getOSTEnv() {
```

Comment:
nit: `getOstEnv` is also fine imo

Reply:
then I'm renaming `initOST` to have same casing